src/main/scala/thurloe/service/ThurloeService.scala

```diff
@@ -158,5 +159,23 @@ trait ThurloeService extends HttpService {
     }
   }
 
-  val keyValuePairRoutes = getRoute ~ getAllRoute ~ queryRoute ~ setRoute ~ deleteRoute
+  val getStatus = path("status") {
```

Comment:
yeah, that's a good idea. we should probably just get rid of the "thurloe" prefix and put those endpoints under something like "keyValues" but that's a story for later